splash/render_options.py

```diff
@@ -169,13 +176,16 @@ def get_headers(self):
             return headers
 
         if not isinstance(headers, (list, tuple, dict)):
-            raise BadOption("'headers' must be either JSON array of (name, value) pairs or JSON object")
+            raise BadOption(
+                "'headers' must be either JSON array of (name, value) pairs or JSON object")
 
         if isinstance(headers, (list, tuple)):
             for el in headers:
-                if not (isinstance(el, (list, tuple)) and len(el) == 2 and all(isinstance(e, basestring) for e in el)):
-                    raise BadOption("'headers' must be either JSON array of (name, value) pairs or JSON object")
-
+                if not (isinstance(el, (list, tuple)) and len(el) == 2 and all(
+                        isinstance(e, six.string_types) for e
+                        in el)):
```

Comment:
I think it is better to move `all(isinstance(e, six.string_types) for e in el)` to a variable - formatting look weird here